internal/cli/project_delete.go

```diff
@@ -0,0 +1,108 @@
+package cli
+
+import (
+	"fmt"
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	"github.com/hashicorp/waypoint/internal/server/gen"
+	"github.com/posener/complete"
+)
+
+type ProjectDeleteCommand struct {
+	*baseCommand
+	flagForce bool
+}
+
+func (c *ProjectDeleteCommand) Run(args []string) int {
+	// Initialize. If we fail, we just exit since Init handles the UI.
+	flagSet := c.Flags()
+	if err := c.Init(
+		WithArgs(args),
+		WithFlags(flagSet),
+		WithNoConfig(),
+	); err != nil {
+		return 1
+	}
+
+	args = flagSet.Args()
+	// Require one argument
+	if len(args) != 1 {
+		c.ui.Output(c.Help(), terminal.WithErrorStyle())
+		return 1
+	}
+	name := args[0]
+
+	// TODO: Use the Interactive terminal, when it's implemented
+	if !c.ui.Interactive() && !c.flagForce {
+		c.ui.Output(
+			"operation interrupted, terminal is not interactive and force not provided",
```

Comment:
I don't think it's worth mentioning an interactive terminal yet until we actually support it. So I'd say let's split out these two error message separately. Then the auto-approve error message could read similar to how we've handled this before:

https://github.com/hashicorp/waypoint/blob/6e6dbd95405adfdf6ee6b75b1f9ce8d014c08ced/internal/cli/server_upgrade.go#L456-L457

```
Deleting a project requires confirmation. 
Rerun the command with '-auto-approve' to continue with deleting the project.
```

Reply:
Oh yes, like an inverse dry-run. I like it!